packages/ember-simple-auth-oauth2/lib/simple-auth-oauth2/authenticators/oauth2.js

```diff
@@ -218,13 +231,15 @@ export default Base.extend({
     @protected
   */
   makeRequest: function(url, data) {
-    return Ember.$.ajax({
-      url:         url,
-      type:        'POST',
-      data:        data,
-      dataType:    'json',
-      contentType: 'application/x-www-form-urlencoded'
-    });
+    var _this = this;
+    var options = {url: url, type: 'POST', data: data, dataType: 'json', contentType: 'application/x-www-form-urlencoded'};
+
+    if(!Ember.isEmpty(_this.clientId)) {
+      var base64ClientId = btoa(_this.clientId + ":");
```

Comment:
Adding a dependency would be not so great IMO. A polyfill would probably be better.